### ISSUE TITLE
Update docstring for hero.wordcloud

### ISSUE THREAD

**Issue by vidyap-xgboost** [jbesomi/texthero#102]
After the discussion on #78 

We should add something like: 

"To reduce blur in the images, width and height should have the same size, i.e the image should be squared"

**Comment by cedricconol** [comment]
Hi @jbesomi @vidyap-xgboost, i'll take this issue.

I also like to add an example in the docstring, but i am not sure how to do this with visualizations. Do you have a suggestion how to do this?

**Comment by jbesomi** [comment]
Thank you Sir: assigned!

We cannot really add a visual example in the docstring, but it's cool you asked.

A very interesting solution that soon or later we might want to implement is to create some visual example with [sphinx-gallery](https://sphinx-gallery.github.io/stable/index.html). Have a look at the documentation of Sphinx-gallery and see if it make sense. For instance, scikit-learn example is using it for the [Examples](https://scikit-learn.org/stable/auto_examples/index.html). 

We might want to implement that. If you want, I can help you with such task, and apparently also @vidyap-xgboost was interested in contributing :)

**Comment by cedricconol** [comment]
Based on the discussion in #78, it seems like width and height does not have to be the same so long as both of them are at least 400. The first example in #78 has the same width and height (200, 200) but it is still blurry.

I think we should rephrase it to:
"To reduce blur in the images, `width` and `height` should be at least 400."

What do you think @vidyap-xgboost @jbesomi? :smiley:

**Comment by cedricconol** [comment]
> Thank you Sir: assigned!
> 
> We cannot really add a visual example in the docstring, but it's cool you asked.
> 
> A very interesting solution that soon or later we might want to implement is to create some visual example with [sphinx-gallery](https://sphinx-gallery.github.io/stable/index.html). Have a look at the documentation of Sphinx-gallery and see if it make sense. For instance, scikit-learn example is using it for the [Examples](https://scikit-learn.org/stable/auto_examples/index.html).
> 
> We might want to implement that. If you want, I can help you with such task, and apparently also @vidyap-xgboost was interested in contributing :)

Yup I'm interested in implementing it through sphinx-gallery, I want to help :smile:

**Comment by vidyap-xgboost** [comment]
> Based on the discussion in #78, it seems like width and height does not have to be the same so long as both of them are at least 400. The first example in #78 has the same width and height (200, 200) but it is still blurry.
> 
> I think we should rephrase it to:
> "To reduce blur in the images, `width` and `height` should be at least 400."
> 
> What do you think @vidyap-xgboost @jbesomi?

I was going to point this out and a little late here, thanks for bringing it up and I agree to your point! But can you also make sure to test for `width=500` and `height=400` and see if any blur is being introduced? Thank you!

**Comment by vidyap-xgboost** [comment]
@jbesomi @cedricconol According to the `numpy docstring guide` as suggested in `CONTRIBUTING.md`:

> The length of docstring lines should be kept to 75 characters to facilitate reading the docstrings in text terminals.

But `wordcloud` seems to have more than 75 characters in its docstring(and yes, its really difficult to read in VS code), can you please take care of that @cedricconol ? I thought I would do it myself but I do not want to interfere with your PR :smiley: , hence asking you.

**Comment by cedricconol** [comment]
> > Based on the discussion in #78, it seems like width and height does not have to be the same so long as both of them are at least 400. The first example in #78 has the same width and height (200, 200) but it is still blurry.
> > I think we should rephrase it to:
> > "To reduce blur in the images, `width` and `height` should be at least 400."
> > What do you think @vidyap-xgboost @jbesomi?
> 
> I was going to point this out and a little late here, thanks for bringing it up and I agree to your point! But can you also make sure to test for `width=500` and `height=400` (you can also test with higher numbers) and see if any blur is being introduced? Follow #96 to see how I tested it on the terminal directly.
> 
> Thank you!

Hi @vidyap-xgboost, I tested `width=500`, `height=400` and `width=1000`, `height=800` following your code in #96. It seems like the blur is gone :smile: 

```python
>>> import texthero as hero
>>> import pandas as pd
>>> df = pd.read_csv('https://github.com/jbesomi/texthero/raw/master/dataset/bbcsport.csv')
>>> import matplotlib
>>> hero.visualization.wordcloud(df['text'],width = 500, height= 400,background_color='White',min_font_size=1)
>>> matplotlib.pyplot.show(block=True)
>>> hero.visualization.wordcloud(df['text'],width = 1000, height= 800,background_color='White',min_font_size=1)
>>> matplotlib.pyplot.show(block=True)
```

Here are the results:
`width=500`, `height=400`
![500400](https://user-images.githubusercontent.com/52105365/87865202-004c2b00-c9a5-11ea-8c31-91bf76521933.png)

`width=1000`, `height=800`
![1000800](https://user-images.githubusercontent.com/52105365/87865206-0cd08380-c9a5-11ea-91af-0fe8359b8f0d.png)

**Comment by cedricconol** [comment]
> @jbesomi @cedricconol According to the `numpy docstring guide` as suggested in `CONTRIBUTING.md`:
> 
> > The length of docstring lines should be kept to 75 characters to facilitate reading the docstrings in text terminals.
> 
> But `wordcloud` seems to have more than 75 characters in its docstring(and yes, its really difficult to read in VS code), can you please take care of that @cedricconol ? I thought I would do it myself but I do not want to interfere with your PR  , hence asking you.

Thanks for the suggestion @vidyap-xgboost, this is noted, it seems like black does not format the docstrings.

**Comment by vidyap-xgboost** [comment]
> > @jbesomi @cedricconol According to the `numpy docstring guide` as suggested in `CONTRIBUTING.md`:
> > > The length of docstring lines should be kept to 75 characters to facilitate reading the docstrings in text terminals.
> > 
> > 
> > But `wordcloud` seems to have more than 75 characters in its docstring(and yes, its really difficult to read in VS code), can you please take care of that @cedricconol ? I thought I would do it myself but I do not want to interfere with your PR  , hence asking you.
> 
> Thanks for the suggestion @vidyap-xgboost, this is noted, it seems like black does not format the docstrings.

Great work @cedricconol ! 
Yes, the docstrings are not formatted by black. Any solution here @jbesomi ?

**Comment by jbesomi** [comment]
Hey @vidyap-xgboost @cedricconol! You are doing great, thank you!

Apparently black does not format docstring. I guess we will need to do it ourselves (@vidyap-xgboost we might explain this better in the CONTRIBUTING.md, what do you think?). If you want to know more: [Black should have an opinion about doc strings](https://github.com/psf/black/issues/144) (still open issue)

**Comment by vidyap-xgboost** [comment]
> Hey @vidyap-xgboost @cedricconol! You are doing great, thank you!
> 
> Apparently black does not format docstring. I guess we will need to do it ourselves (@vidyap-xgboost we might explain this better in the CONTRIBUTING.md, what do you think?). If you want to know more: [Black should have an opinion about doc strings](https://github.com/psf/black/issues/144) (still open issue)

I agree. Explaining in CONTRIBUTING.md sounds good. Thanks for sharing the link!
I would like to work on further changes in documentation and tutorials after the next release since @henrifroese is already working on the documentation. I wouldn't want to interfere and duplicate the work.
